modules/ballerina-http/src/main/ballerina/ballerina/net/http/natives.bal

```diff
@@ -2,6 +2,19 @@ package ballerina.net.http;
 
 @Description { value:"Represents an HTTP request message"}
```

Comment:
Add @Field  annotations with proper details for each struct field

Reply:
Ack